packages/experiments/src/components/Chiclet/Chiclet.types.ts

```diff
@@ -29,18 +29,23 @@ export interface IChicletProps extends React.HTMLAttributes<HTMLElement> {
   /**
    * File title to render for the component.
    */
-  title?: string;
+  title: string;
 
   /**
    * Chiclet size to render
    */
   size?: ChicletSize;
 
   /**
-   * Description to render for the component.
+   * Description to render for the component in place of the url.
    */
   description?: React.ReactElement<JSX.Element>;
 
+  /**
+   * Preview to render for the component.
+   */
+  preview?: React.ComponentType<{ className: string }>;
```

Comment:
Let's actually make this className optional, so 

`preview?: React.ComponentType<{ className?: string }>;`

Reply:
@KevinTCoughlin let us know if this looks right

Reply:
className needs to be a required prop

Reply:
Spoke offline, we're changing this to

```tsx
preview?: React.ComponentType<React.HTMLAttributes<HTMLElement>>;
```